docs/intro/intro.md

```diff
@@ -0,0 +1,21 @@
+# What is Krustlet?
+
+Krustlet is a tool to run WebAssembly workloads natively on Kubernetes. Krustlet acts like a node in
+your Kubernetes cluster. When a user schedules a Pod with certain node tolerations, the Kubernetes
+API will schedule that workload to a Krustlet node, which will then fetch the module and run it.
+
+Krustlet speaks the [kubelet](../topics/glossary.md#kubelet) API, and it will respond to common API
+requests like `kubectl logs` or `kubectl delete`.
+
+If you'd like to learn how to deploy Krustlet on your own cluster (or if you're just getting
+started), follow the [quickstart guide](quickstart.md) for instructions on deploying your first
+Kubernetes cluster.
+
+In order for your application to run on a Kruslet node, it must first be compiled to WebAssembly and
+pushed to a container registry. Krustlet cannot run a Docker container.
```

Comment:
I think this is false. Krustlet can run in a docker container (though that kind of defeats the purpose). I would consider saying that you compile to WebAssembly and push that module to a container registry but it IS NOT a docker image

Reply:
Thank you! I'll see what I can do to clean up this paragraph. The paragraph is referring to the fact that Krustlet cannot run applications like `nginx` or `ubuntu:latest`, which is true. I don't believe I made the claim that Krustlet itself can or cannot run in a container, but if the passage reads that way, I'm more than happy to clear that up. :)